iidm/iidm-xml-converter/src/main/java/com/powsybl/iidm/xml/ShuntXml.java

```diff
@@ -54,35 +77,84 @@ protected void writeRootElementAttributes(ShuntCompensator sc, VoltageLevel vl,
+                    XmlUtil.writeDouble("g", section.getG(), context.getWriter());
+                } catch (XMLStreamException e) {
+                    throw new UncheckedXmlStreamException(e);
+                }
+            });
+            context.getWriter().writeEndElement();
+        } else {
+            throw new PowsyblException(String.format("Unexpected shunt model type for %s: %s", sc.getId(), sc.getModelType().name()));
+        }
+    }
+
     @Override
     protected ShuntCompensatorAdder createAdder(VoltageLevel vl) {
         return vl.newShuntCompensator();
     }
 
     @Override
     protected ShuntCompensator readRootElementAttributes(ShuntCompensatorAdder adder, NetworkXmlReaderContext context) {
-        double bPerSection = XmlUtil.readDoubleAttribute(context.getReader(), "bPerSection");
-        int maximumSectionCount = XmlUtil.readIntAttribute(context.getReader(), "maximumSectionCount");
         int currentSectionCount = XmlUtil.readIntAttribute(context.getReader(), "currentSectionCount");
+        IidmXmlUtil.runUntilMaximumVersion(IidmXmlVersion.V_1_1, context, () -> {
+            double bPerSection = XmlUtil.readDoubleAttribute(context.getReader(), B_PER_SECTION);
+            int maximumSectionCount = XmlUtil.readIntAttribute(context.getReader(), MAXIMUM_SECTION_COUNT);
+            adder.newLinearModel()
+                    .setMaximumSectionCount(maximumSectionCount)
+                    .setbPerSection(bPerSection)
+                    .add();
+        });
         IidmXmlUtil.runFromMinimumVersion(IidmXmlVersion.V_1_2, context, () -> {
             boolean voltageRegulatorOn = XmlUtil.readBoolAttribute(context.getReader(), "voltageRegulatorOn");
             double targetV = XmlUtil.readOptionalDoubleAttribute(context.getReader(), "targetV");
             double targetDeadband = XmlUtil.readOptionalDoubleAttribute(context.getReader(), "targetDeadband");
             adder.setVoltageRegulatorOn(voltageRegulatorOn)
                     .setTargetV(targetV)
                     .setTargetDeadband(targetDeadband);
+            ShuntCompensatorModelType modelType = ShuntCompensatorModelType.valueOf(context.getReader().getAttributeValue(null, MODEL));
+            if (ShuntCompensatorModelType.LINEAR.equals(modelType)) { // default value for linear shunt (always overwritten)
+                adder.newLinearModel()
+                        .setMaximumSectionCount(currentSectionCount + 1) // maximumSectionCount must be > 0
+                        .setbPerSection(Double.MIN_VALUE)
+                        .add();
+            } else if (ShuntCompensatorModelType.NON_LINEAR.equals(modelType)) { // default value for non linear shunt (always overwritten)
+                adder.newNonLinearModel()
+                        .beginSection()
+                            .setSectionNum(currentSectionCount)
+                            .setB(Double.MIN_VALUE)
+                        .endSection()
```

Comment:
Is it a bit strange? Maybe we should create a default Linear Model, like we did for ReactiveLimits? And allow to change the model of a shunt after its creation.

Reply:
It may be a good solution but there are two questions:
 1. What value for `bPerSection` do we want?
 2. Does that mean that we are allowing to modify the shunt model of a built shunt? (we wanted to prevent that at first, given that a modification of shunt model may not really make sense...)

Reply:
Ok, I put another solution that only requires default values for non linear shunt.... what do you think?